draft-bbs-signatures.md

```diff
@@ -577,27 +575,26 @@ Procedure:
 
 ### ProofVerify
 
-ProofVerify checks if a signature proof of knowledge is valid given the proof, the signer's public key, a vector of revealed messages, a vector with the indices of these revealed messages, and the presentation header used in ProofGen.
+This operation checks that a proof is valid for a header, vector of revealed messages (along side their corresponding originally index position when signed) and presentation header against a public key (PK).
```

Comment:
```suggestion
This operation checks that a proof is valid for a header, vector of revealed messages (along side their index corresponding to their original position when signed) and presentation header against a public key (PK).
```
Just a minor suggestion though the original is also completely fine. So really not necessary to use the alternative.